src/common/_modules/function_assets_cdn/autoscaler.tf

```
@@ -0,0 +1,51 @@
+module "function_assets_cdn_autoscale" {
+  source              = "pagopa-dx/azure-app-service-plan-autoscaler/azurerm"
+  version             = "~> 2.0"
+  resource_group_name = module.function_assets_cdn_itn.function_app.resource_group_name
+  location            = var.location_itn
+  app_service_plan_id = module.function_assets_cdn_itn.function_app.plan.id
+  target_service = {
+    function_apps = [
+      { id = module.function_assets_cdn_itn.function_app.function_app.id }
+    ]
+  }
+  scheduler = {
+    normal_load = {
+      minimum = 2
+      default = 2
```

accordingly with Bonus El. prescaling https://github.com/pagopa/io-infra/pull/1687
```suggestion
      minimum = 4
      default = 4
```

---

[f34883a](https://github.com/pagopa/io-infra/pull/1684/commits/f34883ae02409bf9a01440c0b608406ba2409792)